float32 ubication
bool open_button
bool close_button
bool opening_sensor
bool closing_sensor
bool safety_sensor
bool meteorologic_sensor
string state
string action
